### PR TITLE
Add the ability to swap out a topic resource for another

### PR DESCRIPTION
**Pull request by Jonas-C**
Fixes https://github.com/NDLANO/Issues/issues/2948

### CODE REVIEW

File: src/containers/StructurePageBeta/folderComponents/SettingsMenuDropdownType.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ import EditSubjectpageOption from './subjectMenuOptions/EditSubjectpageOption';
 import PublishChildNodeResources from './topicMenuOptions/PublishChildNodeResources';
 import CopyNodeResources from './topicMenuOptions/CopyNodeResources';
 import CopyRevisionDate from './sharedMenuOptions/CopyRevisionDate';
+import SwapTopicResource from './topicMenuOptions/SwapTopicResource';
```
Comment (gunnarvelle) [inline src/containers/StructurePageBeta/folderComponents/SettingsMenuDropdownType.tsx:28]: Et meir korrekt navn er SwapTopicArticle

**Review by haattis — APPROVED** [review]
LGTM

File: src/phrases/phrases-en.ts [hunk 2]
```diff
@@ -1272,6 +1272,12 @@ const phrases = {
     removeLink: 'Remove link',
     jumpToResources: 'Jump to resources',
     jumpToStructure: 'Jump to structure',
+    swapTopicResource: {
+      failed: 'Failed to swap resource',
+      placeholder: 'Resource to swap to',
+      info: 'Swap topic resource',
+      success: 'Topic resource swapped!',
```
Comment (gunnarvelle) [inline src/phrases/phrases-en.ts:1279]:
```suggestion
      failed: 'Failed to swap topic article',
      placeholder: 'Topic article to swap to',
      info: 'Swap topic article',
      success: 'Topic article swapped!',
```

**Review by gunnarvelle — APPROVED** [review]
:100:

**Pull request merged**